src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/ProductSelectionByProducts.tsx

```diff
@@ -41,47 +46,59 @@ export function ProductSelectionByProducts({
     .filter((product) => product.type === selectedProduct.type);
   const [selected, setProduct] = useState(selectedProduct);
 
-  const title = useTextForLanguage(
-    fareProductTypeConfig.configuration.productSelectionTitle,
-  );
+  const title =
+    useTextForLanguage(
+      fareProductTypeConfig.configuration.productSelectionTitle,
+    ) || t(PurchaseOverviewTexts.productSelection.title);
+
+  const subText = (fp: PreassignedFareProduct) => {
+    const descriptionMessage = getTextForLanguage(
+      fp.description ?? [],
```

Comment:
the function allows sends undefined in the case `fp.description` is undefined.

Reply:
Good point, removing the ?? [] 🏂🏼